src/backend/jitsi_magnify/settings.py

```diff
@@ -179,38 +178,25 @@ class Base(Configuration):
     }
 
     REST_FRAMEWORK = {
-        "DEFAULT_AUTHENTICATION_CLASSES": (
-            "rest_framework_simplejwt.authentication.JWTTokenUserAuthentication",
-        ),
         "EXCEPTION_HANDLER": "jitsi_magnify.core.api.exception_handler",
     }
 
-    SIMPLE_JWT = {
-        'ACCES_TOKEN_LIFETIME': timedelta(minutes=60),
-        "ALGORITHM": values.Value("HS256", environ_name="JWT_ALGORITHM"),
-        "SIGNING_KEY": values.SecretValue(
-            environ_name="JWT_JITSI_PRIVATE_SIGNING_KEY",
-            environ_prefix=None
-        ),
-        "AUTH_HEADER_TYPES": ("Bearer",),
-        "AUTH_HEADER_NAME": "HTTP_AUTHORIZATION",
-        'USER_AUTHENTICATION_RULE': 'rest_framework_simplejwt.authentication.default_user_authentication_rule',
-        "AUTH_TOKEN_CLASSES": ("rest_framework_simplejwt.tokens.AccessToken",),
-    }
-
     JWT_CONFIGURATION = {
         "guest_avatar": values.Value(
             "", environ_name="JWT_GUEST_AVATAR", environ_prefix=None
         ),
         "guest_default_password": values.Value(
             "", environ_name="JWT_GUEST_DEFAULT_PASSWORD", environ_prefix=None
         ),
-        "jitsi_url": values.Value(
-            "", environ_name="JWT_JITSI_URL", environ_prefix=None
+        "jitsi_domain": values.Value(
+            environ_name="JWT_JITSI_DOMAIN", environ_prefix=None
         ),
         "jitsi_app_id": values.Value(
             "", environ_name="JWT_JITSI_APP_ID", environ_prefix=None
         ),
+        "jitsi_secret_key": values.SecretValue(
+            environ_name="JWT_JITSI_PRIVATE_SIGNING_KEY", environ_prefix=None
```

Comment:
Maybe uses the same name?
```suggestion
            environ_name="JWT_JITSI_SECRET_KEY", environ_prefix=None
```